stl/inc/flat_set

```diff
@@ -556,10 +557,10 @@ private:
 
     template <class _Ty>
         requires _Keylt_transparent || is_same_v<_Ty, _Kty>
-    size_type _Erase(_Ty&& _Val) {
+    size_type _Erase(const _Ty& _Val) {

```

Comment:
I believe it's safe to use `const auto&` in `_Erase`, as the `auto&&` in `erase` seems only to serve for constraint dispatching.